code/modules/cards/singlecard.dm

```diff
@@ -112,11 +112,11 @@
  * Flips the card over
  *
  * * Arguments:
- * * orientation (optional) - Sets flipped state to CARD_FACEDOWN or CARD_FACEUP if given orientation (otherwise just invert the flipped state)
+ * * is_face_up (optional) - Sets flipped state to CARD_FACEDOWN or CARD_FACEUP if given (otherwise just invert the flipped state)
  */
-/obj/item/toy/singlecard/proc/Flip(orientation)
-	if(!isnull(orientation))
-		flipped = orientation
+/obj/item/toy/singlecard/proc/Flip(is_face_up)
```

Comment:
@tralezab I won't require it for this PR but if you want a very small refactor PR, this should become two separate procs. One for `flip()` and one for `set_face_up` or similar. It shouldn't do both.

Reply:
agree 100%